### DISCUSSION TITLE
CMT2300A - welche Frequenz

### DISCUSSION THREAD

**Discussion by Rustimation** [tbnobody/OpenDTU#1098] [Q&A] [answered]
Hallo Freunde der Erneuerbaren,
ist es egal, welche - erlaubte - Frequenz ich für das CMT Transceivermodul einstelle? Pollt der WR dann nacheinander alle Frequenzen und connected automatisch? Hinter dem "i" Icon steht, dass es eine Viertelstunde dauern kann bis er connected. 
Gibt es eine Default Frequenz mit der die Verbindung besonders schnell steht?
Vielen Dank
Chris

**Comment by bernhard4116** [comment, marked as answer]
Die Default Frequenz des Wechselrichters ist 868,00 MHz, diese hab ich auch in den DTU-Einstellungen hinterlegt - funzt!

**Reply by Rustimation** [reply]
Vielen Dank. Funktioniert.
